modules/dashboard/test/TestPlan.md

```diff
@@ -24,7 +24,7 @@
    Check that you are taken to that page (with the Selection Filter correctly set) when you
    click on the task.
    [Automate Test]
-6. Verify that the issue tracker panel is accessible if and only if a user has the 'issue_tracker_reporter'
+6. Verify that the number of assigned issues in the My Tasks panel is accessible if and only if a user has the 'issue_tracker_reporter'
```

Comment:
I don't think "number of issues [..] is accessible" is clear.. is it that the number of issues shouldn't be displayed? The number should be displayed but not a clickable link? There should be a link but there should be a permission denied error when you click on it? Etc..

Reply:
Ammended.